Form1.cs

```diff
@@ -70,17 +70,17 @@ public partial class Form1 : Form {
 
 		private void applyButton_Click(object sender, EventArgs e) {
 			if(!CheckFolderPath(textbox_path.Text)) {
-				Bad("It seems like the Folder you selected is incorrect. You can go to the properties of Beat Saber in Steam and click on 'Browse Game files' for an easy method to get the correct path");
+				Bad("It seems like the selected folder is incorrect. You can go to the properties of Beat Saber in Steam and click on 'Browse Game files' for an easy method to get the correct path.");
 				return;
 			}
 
 			if(!Regex.IsMatch(textbox_manifest.Text, "[0-9]{19}")) {
-				Bad("The Manifest ID you entered is incorrect. It should be a 19 digit number. Make sure to not accidently enter any spaces");
+				Bad("It seems like the entered Manifest ID is incorrect. It's supposed to be a 19-digit number. Make sure not to enter any spaces by accident.");
 				return;
 			}
 
 			if(Process.GetProcesses().Any(x => x.ProcessName.ToLower() == "steam")) {
-				Bad("Steam seems to be running, please exit it to apply the patch");
+				Bad("Steam seems to be already running, please exit it to apply the patch.");
```

Comment:
"Already running" would be wrong here - We arent trying to start Steam, merely telling the user to end it